okhttp/src/main/java/io/grpc/okhttp/OkHttpClientStream.java

```diff
@@ -195,7 +200,7 @@ public TransportState(
         AsyncFrameWriter frameWriter,
         OutboundFlowController outboundFlow,
         OkHttpClientTransport transport) {
-      super(maxMessageSize, statsTraceCtx);
+      super(maxMessageSize, statsTraceCtx, null /* transportTracer */);
```

Comment:
optional:

The new linter nags about comments not in the form:

`/*transportTracer=*/ null`